flagsmith-core.ts

```diff
@@ -131,11 +131,22 @@ const Flagsmith = class {
             })
     };
 
+    _onChange: IInitConfig['onChange'] = (previousFlags, params, loadingState) => {
```

Comment:
onChange was initialised on flagsmith.init, this means that triggers weren't setup if flagsmith.init wasn't called in an SSR scenario. Now this is done on construction.